firefox-ios/Client/Frontend/ContextualHint/ContextualHintViewProvider.swift

```diff
@@ -115,14 +114,7 @@ class ContextualHintViewProvider: ContextualHintPrefsKeysProvider, SearchBarLoca
     }
 
     var isActionType: Bool {
```

Comment:
If this is always false, should we clean that up?

Reply:
Are we sure we will never have actionable CFRs again? Then yes, we should definitely clean this up.

Reply:
I mean I don't know for sure, but I don't like keeping code around for "what ifs". We can bring it back if needed? Maybe do that more easily we can create a separate PR for this only?

Reply:
That sounds like a great idea. Will create another PR after this then.